### PR TITLE
12345678 | master

### PR DESCRIPTION
**Pull request by yanan**
https://...
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase [hunk 1]
```diff
@@ -0,0 +1,70 @@
+@component-name = "portal-smoke"
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase:1]: portal-user-management-and-portal-configuration

File: portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+definition {
+
+	property testray.main.component.name = "Smoke";
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase:4]: Roles

File: portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+		else {
+			PagesAdmin.tearDownCP();
+
+			Site.tearDownCP();
+
+			WebContent.tearDownCP();
+		}
+	}
+
+	@description = "Add tests for LPS-148855, ensure guest group roles are inherited by the Guest user."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase:25]: This is a test for LPS-148855. The default site role shouldn't be inherited by Guest.

File: portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+	@priority = "5"
+	test CheckGuestCannotViewDocumentViewers {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase:27]: DefaultSiteRoleShouldNotBeInheritedByGuest

File: portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+		User.firstLoginPG();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase:28]: Move to setup block

File: portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+		JSONRole.addRegularRole(
+			roleKey = "Document Viewer",
+			roleTitle = "Document Viewer");
+
+		JSONGroup.addGroup(groupName = "Site Name");
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase:35]: Missing assign site to regular role

**Comment by yanan** [comment]
Closing this pull request, resend in https://github.com/Tim-Cao/liferay-portal/pull/686

**Pull request closed without merging**